metadata-ingestion/src/datahub/ingestion/source/sql/sql_generic_profiler.py

```diff
@@ -74,7 +74,7 @@ def generate_profiles(
         for request in table_level_profile_requests:
             profile = DatasetProfile(
                 timestampMillis=int(datetime.now().timestamp() * 1000),
-                columnCount=len(request.table.columns),
+                columnCount=request.table.column_count,
```

Comment:
What if this is None?

maybe request.table.column_count or 0?

Reply:
it can't be None, it either 0 or something. It's type is not optional.